.ci/scripts/distribution/analyse-java.sh

```diff
@@ -23,12 +23,12 @@ else
   fi
 
   if [ "${GIT_BRANCH}" == "master" ] || [ "${GIT_BRANCH}" == "develop" ]; then
-    TARGET_BRANCH="master"
+    TARGET_BRANCH="${GIT_BRANCH}"
   else
     TARGET_BRANCH="develop"
+    PROPERTIES+=("-Dsonar.branch.target=${TARGET_BRANCH}")
   fi
 
-  PROPERTIES+=("-Dsonar.branch.target=${TARGET_BRANCH}")
```

Comment:
Can you explain what this fixes? It's unclear to me

Reply:
This doesn't fix anything, it was just unnecessary unless you were comparing for a PR.